tests/robot-tests/tests/admin_and_public/bau/prerelease_visibility.robot

```diff
@@ -27,7 +27,8 @@ Verify release summary
 Upload subject
     [Tags]  HappyPath
     user clicks link  Data and files
-    user waits until h2 is visible  Add data file to release
+    user waits for page to finish loading
```

Comment:
This shouldn't be necessary here, as we have a wait immediately after it.

Reply:
Without this keyword the pipeline seemed to consistently fail on: Element 'xpath://*[contains(@class,"govuk-accordion__section-button") and text()="UI test subject"]' did not appear in 30 seconds.